umad-09-invoice.md

```diff
@@ -0,0 +1,143 @@
+# UMAD-09 Invoice
+
+An UMA invoice is a data structure that describes payments that can be coordinated using UMA and
+provides a way for a payment sender to have proof of payment–without having to use the existing
+LNURLPRequest/Response handshake flow.
+
+UMA invoices allow a payment recipient to specify an amount they want to receive in any currency,
+along with some details on how to pay. This is useful in many cases where the recipient is the
+initiator of a payment. For example, merchant checkout flows, a bill from any vendor or service
+provider, etc.
+
+## Invoice Format
+
+The full structure of the invoice is:
+
+```raw
+{
+    ReceiverUma: string, 
+    // Receiving UMA address
```

Comment:
nit: All of the other code examples in the spec put doc comments above fields like JS-style fields.